src/lib/sidebar/EntireScheme.svelte

```diff
@@ -112,63 +125,67 @@
   function loadFile(text: string) {
     try {
       // TODO Should we prompt before deleting the current scheme?
-      gjScheme.set(backfill(JSON.parse(text)));
+      gjSchemeCollection.set(backfill(JSON.parse(text)));
       errorMessage = "";
     } catch (err) {
       errorMessage = `Couldn't load scheme from a file: ${err}`;
     }
   }
 
-  $: numErrors = $gjScheme.features.filter(
+  $: numErrors = $gjSchemeCollection.features.filter(
     (f) => interventionWarning(schema, f) != null
   ).length;
 </script>
 
-{#if $mode.mode == "list"}
-  <CollapsibleCard label={$gjScheme.scheme_name ?? "Untitled scheme"}>
-    <TextInput label="Scheme name" bind:value={$gjScheme.scheme_name} />
-
-    <ErrorMessage {errorMessage} />
-
-    <FileInput label="Load from GeoJSON" id="load-geojson" {loadFile} />
-
-    <SecondaryButton on:click={exportToGeojson}>
-      Export to GeoJSON
-    </SecondaryButton>
-
-    <div>
-      <WarningButton
-        on:click={() => (displayClearAllConfirmation = true)}
-        disabled={$gjScheme.features.length == 0}
+{#if scheme}
```

Comment:
Just to reduce nesting and the git diff, could we combine? `{#if scheme && $mode.mode == "list}`